src/Skeletor/Console/CreateProjectCommand.php

```diff
@@ -42,7 +42,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->setupCommand();
 
         //Start process in the background
-        $process = new Process('php skeletor package:show --no-ansi');
+        $process = new Process('skeletor package:show --no-ansi');
```

Comment:
Is it right that this is being changed back? This was a bugfix you did last time to sort the issue with the packagist API, right?